packages/zoe/src/zoeService/zoe.js

```diff
@@ -181,12 +222,20 @@ const makeZoeKit = (
     getInvitationDetails,
   } = makeInvitationQueryFns(invitationIssuer);
 
+  const getConfiguration = () => {
+    return harden({
+      feeIssuerConfig,
+      zoeFeesConfig,
+      meteringConfig,
+    });
+  };
+
   /** @type {ZoeServiceFeePurseRequired} */
   const zoeService = Far('zoeServiceFeePurseRequired', {
     install,
     startInstance,
     offer,
-    makeFeePurse,
+    makeFeePurse: async () => makeFeePurse(),
```

Comment:
No objection, but curious why this change?

Reply:
I needed a version of `makeFeePurse` that returned a promise (the external version) and a version that was synchronous. This is converting to the external version.

Reply:
Good. Resolving.